modules/nf-core/samtools/faidx/main.nf

```diff
@@ -9,11 +9,13 @@ process SAMTOOLS_FAIDX {
 
     input:
     tuple val(meta), path(fasta)
+    tuple val(meta2), path(fai)
 
     output:
-    tuple val(meta), path ("*.fai"), emit: fai
-    tuple val(meta), path ("*.gzi"), emit: gzi, optional: true
-    path "versions.yml"            , emit: versions
+    tuple val(meta), path ("*.{fa,fasta}") , emit: fa , optional: true
```

Comment:
Why outputting fa?
any difference between the input?

Reply:
Yeah, when you extract sequences using faidx, output is written in fasta format

Reply:
but...you didn't pass the second argument to samtools...so what is the purpose of this?